public/app/features/logs/components/logParser.ts

```diff
@@ -57,10 +57,6 @@ function shouldRemoveField(field: Field, index: number, row: LogRowModel) {
   if (field.name === 'id' || field.name === 'tsNs') {
     return true;
   }
-  // entry field which we are showing as the log message
-  if (row.entryFieldIndex === index) {
```

Comment:
The line may get very long and we may end up with lots of annoying duplication in the UI. Maybe we can truncate it?

<img width="1438" alt="Screenshot 2023-02-24 at 10 31 33" src="https://user-images.githubusercontent.com/745532/221144240-55c5b9b7-1800-447d-9ca2-0aab3c95c5f3.png">

Reply:
I added a scrollable element that will be put in if the length of the string is over 3x as wide as the element it would go in. Right now it does not respond to screen resize events. Let me know what you think of this solution.